Transport/Http/HttpTransport.php

```diff
@@ -53,6 +54,6 @@ public function reject(Envelope $envelope): void
 
     public function send(Envelope $envelope): Envelope
     {
-        return $envelope;
+        throw Exception('Problème to send on your producer');
```

Comment:
Non il n'y a pas de "problem". C'est juste non supporté ^^

Plutôt mettre "This transport does not support the send action"